packages/edit-widgets/src/blocks/legacy-widget/edit/widget-preview.js

```diff
@@ -15,7 +15,7 @@ function WidgetPreview( { widgetAreaId, attributes, hidden, ...props } ) {
 	const HEIGHT_MARGIN = 20;
 	const [ height, setHeight ] = useState( DEFAULT_HEIGHT );
 	const iframeRef = useRef();
-	const currentUrl = document.location.href;
+	const currentUrl = `${ document.location.protocol }//${ document.location.host }/wp-admin/themes.php?page=gutenberg-widgets`;
```

Comment:
This won't work if WordPress is installed in a subdirectory e.g. `example.com/blog/wp-admin`.

Reply:
Ahh! Good point!

Reply:
I know we have had this issue elsewhere... I think we work around it using an editor setting.

Reply:
I see we have `getWPAdminURL()` in `@wordpress/editor`. Maybe that's what we want 😛